packages/eslint-plugin/src/rules/no-empty-styled-expression/__tests__/rule.test.ts

```diff
@@ -0,0 +1,65 @@
+import type { RuleTester } from 'eslint';
+
+import { typeScriptTester as tester } from '../../../test-utils';
+import { noEmptyStyledExpressionRule } from '../index';
+
+type InvalidTestCase = Omit<RuleTester.InvalidTestCase, 'errors'>;
+
+const createInvalidTestCases = (tests: InvalidTestCase[]) =>
+  tests.map((t) => ({
+    ...t,
+    errors: [{ messageId: 'unexpected' }],
+  }));
+
+tester.run('no-styled-tagged-template-expression', noEmptyStyledExpressionRule, {
+  valid: [
+    `
+      import { styled } from 'styled';
+
+      styled.div({
+        color: blue
+      });
+    `,
+    `
+      import { styled } from 'styled';
+
+      styled.span({
+        color: blue
+      });
+    `,
+  ],
```

Comment:
Can you please add a test case where you pass something that isn't an object expression?

Reply:
Yeah e.g. arrays

https://github.com/atlassian-labs/compiled/blob/master/packages/react/src/styled/index.ts#L64

Perhaps

```
styled.div([])
```

might be another loophole we need to address too?

And maybe booleans like styled.div(false && {}), but that's probably getting a bit obscure

https://github.com/atlassian-labs/compiled/blob/master/packages/react/src/types.ts#L35